aes/decryptor.go

```diff
@@ -46,54 +51,67 @@ func parse(value string) (*encryptedValue, error) {
 	return &encryptedValue{data, iv, tag, datatype}, nil
 }
 
-// Decrypt takes a sops-format value string and a key and returns the decrypted value.
-func (c Cipher) Decrypt(value string, key []byte, additionalAuthData []byte) (interface{}, error) {
+// Decrypt takes a sops-format value string and a key and returns the decrypted value and a stash value
+func (c Cipher) Decrypt(value string, key []byte, path string) (plaintext interface{}, stash interface{}, err error) {
```

Comment:
Three return values is starting to become a lot. At that point it might be better to wrap them in a struct.

It looks like `stash` is always type `stashData`, does the type need to be `interface{}`?